Makefile

```diff
@@ -98,5 +98,8 @@ mo:
 	cd example && python ./manage.py compilemessages
 	cd pynotify && django-admin compilemessages
 
-makemigrations:
```

Comment:
Whoops, `makemigrations` was actually `migrate`. The target was missing completely.